schemas/oic.linkstatus-rw-schema.json

```diff
@@ -0,0 +1,46 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description" : "Copyright (c) 2018 Open Connectivity Foundation, Inc. All rights reserved.",
+    "id": "http://openconnectivityfoundation.github.io/core/schemas/oic.linkstatus-rw-schema.json#",
+    "title": "Linkstatus Resource Type",
+    "definitions": {
+      "linkstatus-properties": {
+        "properties": {
+          "linkstate": {
+            "type": "string",
+            "enum": [ "init", "active", "error", "timeout" ],
+            "readOnly": true,
+            "description": "The state of the dynlink"
+          },
+          "laststatus": {
+            "type": "number",
+            "enum": [200, 201, 400, 403, 404, 405, 406, 408, 409, 410, 415, 418, 429, 500, 501, 502, 503, 504],
+            "readOnly": true,
+            "description": "Last status code in http equivalent"
+		      },
+          "endtime": {
+            "type": "string",
+            "format": "date",
+            "readOnly": true,
+            "description": "Time at which the link times out if it remains idle"
+		      },
+          "maxwait": {
+            "type": "number",
+            "readOnly": false,
+            "description": "Maximum idle seconds until timeout error"
+		      },
+          "linkinstance": {
+            "type": "string",
```

Comment:
The CR says this is a "number" but I think "integer" would be better.  Either way, I don't think it should be a string.